README.md

```diff
@@ -85,96 +84,69 @@ public void login(String username, String password, String regTimeout) {
    endpoint.login(username, password, regTimeout);
```

Comment:
We can remove this function since in the latest beta version we have removed this method